src/map/utils/fishingutils.cpp

```diff
@@ -2346,7 +2346,21 @@ namespace fishingutils
             if (sub > 0)
             {
                 ItemPoolWeight -= sub;
-                NoCatchWeight += sub;
+                // These are truly weights, so we don't need any particular totals
+                // NoCatchWeight += sub;
```

Comment:
As you pointed out, you've found `no evidence that "decreasing the chance of items/mobs" should simply shift that chance to getting nothing` which means you also have not found evidence of the contrary either, so change this back, please.